sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -249,7 +253,11 @@ export class ManagementClient extends LinkEntity {
    * @param {string} type - The type of entity requested for. Valid values are "eventhub", "partition"
    * @param {string | number} [partitionId] - The partitionId. Required only when type is "partition".
    */
-  private async _makeManagementRequest(type: "eventhub" | "partition", partitionId?: string | number): Promise<any> {
+  private async _makeManagementRequest(
+    type: "eventhub" | "partition",
+    options?: RequestOptions,
+    partitionId?: string | number
```

Comment:
This order seems off. The `type` and `partitionId` seem to be related. How about making `options` the first parameter here? It can be made mandatory.

Reply:
Everywhere(Event Hubs, Service bus) we are using `options` as a optional parameter, just to make order right, I'm not sure of making `options` as a mandatory parameter.

What do you think?

Reply:
In that case lets make this `_makeManagementRequest` simpler. I believe the `type` and `partitionId` is used to make the `request` object. Let's pull that out to the caller and have the `_makeManagementRequest` taken in just the `request` and `options`.

That would be a much cleaner design

Reply:
Just made a small change, other than looks good. Approved